docker/install-scripts/azure-cli.sh

```diff
@@ -13,6 +13,16 @@ while [ $# -gt 0 ]; do
   shift
 done
 
-pip --no-cache-dir install azure-cli==${VERSION}
+curl -sL https://packages.microsoft.com/keys/microsoft.asc |
+    gpg --dearmor |
+    tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null
+
+    AZ_REPO=$(lsb_release -cs)
+echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $AZ_REPO main" |
+    tee /etc/apt/sources.list.d/azure-cli.list
+
+  apt update
+  apt-get install azure-cli=${VERSION}-1~$AZ_REPO
```

Comment:
I would have added `-y` here but (assuming there are no dependencies to instal today) that is a matter of taste. This would mean the command would continue to work if MS would add additional dependencies, e.g. some c lib for which they depend on an older version.